src/components/SectionSwap.tsx

```diff
@@ -44,10 +45,15 @@ const SectionSwap = ({ type }: { type: string }) => {
   const brand = purse?.brand;
 
   const handleValueChange = (value: bigint) => {
-    if (type === SectionSwapType.FROM) {
-      setFromAmount(AmountMath.make(brand, value));
-    } else {
-      setToAmount(AmountMath.make(brand, value));
+    switch (type) {
+      case SectionSwapType.FROM:
+        setFromAmount(AmountMath.make(brand, value));
+        break;
+      case SectionSwapType.TO:
+        setToAmount(AmountMath.make(brand, value));
+        break;
+      default:
+        break;
```

Comment:
I think you can drop the default since you're handling every value case. TS can verify this. if something is complaining it might be eslint or TS config needs tweaking.

Reply:
Done